frontend/components/queries/QueryProgressDetails/QueryProgressDetails.jsx

```diff
@@ -51,12 +53,11 @@ const QueryProgressDetails = ({ campaign, className, onRunQuery, onStopQuery, qu
   return (
     <div className={`${baseClass} ${className}`}>
       <div className={`${baseClass}__wrapper`}>
-        <span>
-          <b>{hostsCount.total}</b>&nbsp;of&nbsp;
-          <b>{totalHostsCount} Hosts</b>&nbsp;Returning&nbsp;
-          <b>{totalRowsCount} Records&nbsp;</b>
-          <em>({hostsCount.failed} failed)</em>
-        </span>
+        <div className={`${baseClass}__text-wrapper`}>
+          <span className={`${baseClass}__text-online`}>Online - {metrics.OnlineHosts} hosts returning {totalRowsCount} results</span>
+          <span className={`${baseClass}__text-offline`}>Offline - {metrics.OfflineHosts} hosts returning 0 results</span>
+          <span className={`${baseClass}__text-error`}>Failed - {hostsCount.failed} hosts returning {errors ? errors.length : '0'} errors</span>
+        </div>
```

Comment:
nit: Let's make sure the pluralization is correct in cases where there is only 1 host/error